### PR TITLE
calibration

### PR DESCRIPTION
**Pull request by Towflos**
#22 added mag_radius and acc_radius to parameters, added Trigger service to request calibration data

### CODE REVIEW

File: bno055/sensor/SensorService.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
 from rclpy.qos import QoSProfile
 from sensor_msgs.msg import Imu, MagneticField, Temperature
 from std_msgs.msg import String
+from example_interfaces.srv import Trigger
```
Comment (flynneva) [inline bno055/sensor/SensorService.py:43]: @Towflos we should probably add `example_interfaces` as a dependency to the `package.xml` then, right?

**Comment by flynneva** [comment]
@Towflos wow this is a great feature! adding a service to calibrate the IMU.  Do you think you could add a short section to the `README` on you expect people to use the service?

**Comment by Towflos** [comment]
@flynneva the service only reads the calibration values and returns them in a message. I added the command to call the service to the ``README``.

**Comment by flynneva** [comment]
@Towflos great stuff again, well done 🍻

**Pull request merged**